### DISCUSSION TITLE
Issues with Current Scaling and Voltage Limits

### DISCUSSION THREAD

**Discussion by Maguixx** [pybamm-team/liionpack#302] [Q&A]
Hello!

I am currently working on reproducing an experiment from an article using Liionpack and have encountered several issues regarding current scaling and voltage limits.

Problem Description:
Current Scaling:
To match the article's current per cell "8p1s" , I modified the experiment as follows:

experiment = pybamm.Experiment(
    [
        "Discharge at 240A for 242 seconds",
        "Charge at 40A for 1468 seconds",
    ],
    period="2 seconds",
)

Voltage Limit Issues:
When running the above experiment, I encountered the "low voltage limit reached" error. To circumvent this, I adjusted the experiment's duration:

experiment = pybamm.Experiment(
    [
        "Discharge at 240A for 14 seconds",
        "Charge at 40A for 26 seconds",
    ],
    period="2 seconds",
)

Despite these adjustments, there is a significant difference in the simulation time compared to the article, although the shape of the resulting graphs is not vastly different.

Request for Assistance:
I am seeking advice on the following points:

How can I appropriately scale the currents to match the per-cell current requirements without encountering voltage limit issues?
Are there any parameter adjustments or configurations within Liionpack that could help achieve a closer match to the article's experimental conditions while maintaining stability in the simulation?
Any guidance or suggestions from the community would be greatly appreciated.

Thank you!

**Comment by TomTranter** [comment]
Which paper are you referring to? Can you email me tom@ionworks.com
